code/src/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Are you using normalize.css to make the project adaptable to a different web browser? Am I understanding it properly?

Reply:
Yes, Normalize.css provides cross-browser consistency, similar to the way Reset CSS works.